src/cascadia/TerminalConnection/EchoConnection.cpp

```diff
@@ -42,12 +42,9 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
     {
         rows;
         columns;
-
-        throw hresult_not_implemented();
     }
 
     void EchoConnection::Close()
     {
-        throw hresult_not_implemented();
```

Comment:
Were these supposed to be removed?

Reply:
Yea, if left in they crash the test when the connection gets torn down 😬